src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py

```diff
@@ -395,14 +395,14 @@ def evaluate(
 
         # drop input columns
         evaluator_result_df = evaluator_result_df.drop(
-            columns=[col for col in evaluator_result_df.columns if col.startswith(Prefixes._INPUTS)]
+            columns=[col for col in evaluator_result_df.columns if str(col).startswith(Prefixes._INPUTS)]

```

Comment:
this is a good conner case can be covered by unit test. if takes time, pls create an ado for now.

Reply:
Fixed